src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/DoNotAddFinalizerToMemoryManagerTDerivedTypes.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using Analyzer.Utilities;
+using Analyzer.Utilities.Extensions;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Microsoft.NetCore.Analyzers.Runtime
+{
+    /// <summary>
+    /// CA2014: Do not add finilazer for MemoryManaged&lt;T&gt; derived types.
+    /// </summary>
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
```

Comment:
Not sure if it need to support VisualBasic too

Reply:
Yep, it should support VB